### DISCUSSION TITLE
datasource container cloudera.impala

### DISCUSSION THREAD

**Discussion by WrettySV** [testcontainers/testcontainers-java#6419] [Q&A] [answered]
Cannot find the right container for cloudera.impala and the right driver class name. 
I found this docker image cloudera/impala-dev:minimal, so the datasource url looks like this jdbc:tc:cloudera/impala-dev:minimal, but I have a mistake that driver name  set in configuration file (org.testcontainers.jdbc.ContainerDatabaseDriver) does not fit to this url.

**Comment by eddumelendez** [comment, marked as answer]
There is no specific module for cloudera so there is no testcontainers jdbc url for it. You should use `GenericContainer`

**Reply by WrettySV** [reply]
Thank you so much for replying. Yes, generic container, but I have necessity to point driver class name in property. Should I point org.testcontainers.jdbc.ContainerDatabaseDriver?

**Reply by eddumelendez** [reply]
You should use the cloudera jdbc driver classname
